libcobblersignatures/signatures.py

```diff
@@ -125,10 +118,7 @@ def _importsignaturesfile(self, filepath):
                          by the Python ``open()`` function.
         """
         with open(filepath) as f:
```

Comment:
Would be better to only open in read mode.

Reply:
Done